apps/web/playwright/fixtures/users.ts

```diff
@@ -330,7 +331,19 @@ export const createUsersFixture = (page: Page, workerInfo: WorkerInfo) => {
       await page.goto("/auth/logout");
     },
     deleteAll: async () => {
+      const emailMessageIds: string[] = [];
       const ids = store.users.map((u) => u.id);
+      for (const user of store.users) {
+        const emailMessages = await emails.search(user.email);
+        if (emailMessages && emailMessages.count > 0) {
+          emailMessages.items.forEach((item) => {
+            emailMessageIds.push(item.ID);
+          });
+        }
+      }
+      for (const id of emailMessageIds) {
+        await emails.deleteMessage(id);
+      }
```

Comment:
@leog @shivamklr I think this is why I get so many errors running e2e locally on production builds.

<img width="1062" alt="image" src="https://github.com/calcom/cal.com/assets/3504472/f09ab57f-3a41-408d-a4dd-05f63c0e406d">

Can we make this optional? Only check for mailhog if it's enabled. I spent quite a while figuring out why all my tests were failing locally. At least we should document and update the README with instructions.

Reply:
Sure. I will take a look and update the users fixture.